serve.py

```diff
@@ -103,34 +102,23 @@ async def goto_public_dashboard(request):
 
 @app.route("/dandiset/<dataset:int>", methods=['GET', 'HEAD'])
 async def goto_dandiset(request, dataset):
-    """Redirect to gui with retrieved folder ID
+    """Redirect to gui with dandiset identifier
     """
-    req = requests.get(
-        f"https://girder.dandiarchive.org/api/v1/dandi/{dataset:06d}")
-    if req.reason == 'OK':
-        json_info = req.json()
-        if json_info is not None:
-            id = json_info['_id']
-            url = f"https://gui.dandiarchive.org/#/dandiset/{id}"
-            if request.method == "HEAD":
-                return response.html(None, status=302, headers=make_header(url))
-            return response.redirect(url)
-    return response.text(f"dandi:{dataset:06d} not found.", status=404)
+    url = f"https://gui.dandiarchive.org/#/dandiset/{id}"
```

Comment:
Isn't the relevant variable `dataset`? 
```suggestion
    url = f"https://gui.dandiarchive.org/#/dandiset/{dataset:06d}"
```